src/ctransaction.cpp

```diff
@@ -224,7 +224,9 @@ char * CTransaction::bytes() {
         qint16 plen = qToBigEndian(pParameters[i]->length());
         memcpy(b+24+offset, &plen, 2);
 
-        qint16 sval, ival;
+        qint16 sval;
+        qint32 ival;
```

Comment:
Wanted to double check that this change was valid. It was trying to read 4 bytes from a var that only stored two... so I updated the type, which I believe is correct.

Reply:
Yes, this was a bug for sure.

Reply:
Awesome, we should try to get the bug fixes into the main branch. Then the leftovers I can keep in a QT5 branch for now.